### PR TITLE
Homework7: Valeria Kudelko

### PR DESCRIPTION
**Pull request by ValeriaKudelko**

### CODE REVIEW

File: thirdfile.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+        bulls, cows = check_guess(secret_number, guess)
+        print(f"Быков: {bulls}, коров: {cows}")
+
+        if bulls == 4:
+            print(f"Вы отгадали число {secret_number} за {attempts} попыток.")
+            break
+
+
+if __name__ == "__main__":
```
Comment (Stml89) [inline thirdfile.py]: you do not need this line, pls remove it

File: thirdfile.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+# нравится доводить вещи до совершенства, то необходимо расположить их от
+# меньшего к большему, чтобы каждая статуя была больше предыдущей ровно
+# на 1. Для этого Вам могут понадобиться дополнительные статуи.
+# Определите количество отсутствующих статуй.
+# Пример:
+# Для статуй = [6, 2, 3, 8] результат должен быть = 3. Иными словами,
+# у Вас отсутствуют статуи размеров 4, 5 и 7.
+
+statues = [6, 2, 3, 8]
+statues.sort()
```
Comment (Stml89) [inline thirdfile.py]: do not need to sort it, just find MIN and MAX values and based on this [MIN..MAX] find missing numbers

**Review by Stml89 — COMMENTED** [review]
Looks good, fix small issues and merge it

File: thirdfile.py (hunk 1 above)
Comment (ValeriaKudelko) [reply thirdfile.py]: Its done

File: thirdfile.py (hunk 2 above)
Comment (ValeriaKudelko) [reply thirdfile.py]: corrected

**Pull request merged**